packages/fxa-auth-server/lib/payments/paypal.ts

```diff
@@ -345,6 +345,10 @@ export class PayPalHelper {
     options: TransactionSearchOptions
   ): Promise<TransactionSearchResult[]> {
     const results = await this.client.transactionSearch(options);
+    if (!(results.L instanceof Array)) {
+      return [];
+    }
```

Comment:
We should check for empty array in `attemptInvoiceProcessing` and early return as well.